websites/D/dアニメストア/presence.ts

```diff
@@ -1,36 +1,29 @@
-{
-	const presence = new Presence({
-			clientId: "611012705306017792",
-		}),
-		strings = presence.getStrings({
-			play: "presence.playback.playing",
-			pause: "presence.playback.paused",
-		});
+const presence = new Presence({
+  clientId: "611012705306017792"
+});
 
-	presence.on("UpdateData", async () => {
-		if (
-			location.pathname.startsWith("/animestore/sc_d_pc") &&
-			document.querySelector("#video")
-		) {
-			const video: HTMLVideoElement = document.querySelector("#video"),
-				isPlaying = !video.paused,
-				presenceData: PresenceData = {
-					details: `${document.querySelector(".backInfoTxt1").textContent} - ${
-						document.querySelector(".backInfoTxt2").textContent
-					}`,
-					state: document.querySelector(".backInfoTxt3").textContent,
-					largeImageKey: "danime",
-					smallImageKey: isPlaying ? "play" : "pause",
-					smallImageText: isPlaying
-						? (await strings).play
-						: (await strings).pause,
-					startTimestamp:
-						Math.floor(Date.now() / 1000) - Math.floor(video.currentTime),
-				};
+const strings = await presence.getStrings({
+  play: "general.playing",
+  pause: "general.paused",
+});
 
-			if (!isPlaying) delete presenceData.startTimestamp;
-
-			presence.setActivity(presenceData);
-		}
-	});
-}
+presence.on("UpdateData", async () => {
+  if (
+    location.pathname.startsWith("/animestore/tp_mn") &&
+    document.querySelector("#dplayer_video_box video")
+  ) {
+    const video = document.querySelector("#dplayer_video_box video"),
+          isPlaying = !video.paused,
+          presenceData = {
+            details: `${document.querySelector(".backInfoTxt1").textContent} - ${document.querySelector(".backInfoTxt2").textContent}`,
+            state: document.querySelector(".backInfoTxt3").textContent,
+            largeImageKey: "https://i.imgur.com/6sp3k8m.png",
+            smallImageKey: isPlaying ? "play" : "pause",
```

Comment:
Imgur URLs are used instead of Discord asset names here. Also, put the assets in an enum:
https://github.com/PreMiD/Presences/blob/3aa7e3fb8e54999012de1d1528f37bc907ae4213/websites/C/Crunchyroll/presence.ts#L52-L58